scripts/ci/build_emscripten.sh

```diff
@@ -34,58 +34,101 @@
 
 set -ev
 
-if test -z "$1"; then
-	BUILD_DIR="emscripten_build"
-else
-	BUILD_DIR="$1"
-fi
-
 apt-get update
```

Comment:
I intentionally left the apt-get, since we plan to remove it in https://github.com/ethereum/solidity/pull/13527/files

Reply:
#13527 is now merged, so please rebase.